cmd/run.go

```diff
@@ -222,6 +232,11 @@ func DefaultConfig() *Config {
 			Enabled: false,
 			Addr:    ":3001",
 		},
+		Metrics: MetricsConfig{
+			Enabled:  false,
+			Protocol: "grpc",
+			Endpoint: "localhost:4317",
```

Comment:
To be consistent with the other defaults do you mind if we bind to `0.0.0.0:4317` instead of `localhost`?